typings/cordova.d.ts

```diff
@@ -0,0 +1,7 @@
+interface Cordova {
+    getAppVersion: {
+        getVersionCode: Function;
```

Comment:
Please dont use `Function` only you dont know the signature?
You know the callback and signature, so please add it.